internal/handlers/common/params.go

```diff
@@ -192,3 +192,47 @@ func getBinaryParams(fieldValue any, maskValue any) (types.Binary, types.Binary,
 
 	return fieldBinary, maskBinary, nil
 }
+
+// getTypeAliasByCode returns type alias and error by given type code.
```

Comment:
Somewhere should be a description what "type alias" and "type code" are.

One option could be to define custom types for them:
```go
// typeCode ...
type typeCode int32
```

It then allows us to define constants and their string representation by using stringer code generator